app_integrations/apps/salesforce.py

```diff
@@ -327,24 +329,23 @@ def _list_log_files(self):
             ]
         }
         """
-        log_files = []
-        for event_type in self._SALESFORCE_EVENT_TYPES:
-            event_type_format = 'AND+EventType+=+\'{}\''.format(event_type)
-            url = self._SALESFORCE_QUERY_URL.format(
-                instance_url=self._instance_url,
-                api_version=self._latest_api_version,
-                query=self._SALESFORCE_QUERY_FILTERS,
-                start_time=self._SALESFORCE_CREATE_AFTER.format(self._last_timestamp),
-                event_type=event_type_format
-            )
-            success, response = self._make_get_request(url, self._auth_headers)
-            if not success:
-                LOGGER.error('Failed to get a list of log files.')
-                return log_files
+        event_type_format = 'AND+EventType+=+\'{}\''.format(self._type)
```

Comment:
you can probably just move this var declaration right to the `event_type` keyword argument right?